wagtailcache/cache.py

```diff
@@ -183,6 +198,13 @@ def process_response(
             cache_key = learn_cache_key(
                 request, response, timeout, None, cache=self._wagcache
             )
+
+            # Track cache keys
+            uri = _clean_uri(request.build_absolute_uri())
+            keyring = self._wagcache.get("keyring", {})
```

Comment:
Not necessary, but an idea for a cool feature: show the list of URLs in the keyring on the wagtail cache admin page. Could possibly include a little button to clear each individual page from the cache.

Not necessary at all... we can add it in later. But it would be a good way to show off the feature :)

Reply:
https://github.com/coderedcorp/wagtail-cache/pull/22#discussion_r597785700